# program produce unexpected result

def add(a,b):
  # return a * b logical error
  return a + b

print(add(1,2))